modules/newFunc.js

```diff
@@ -0,0 +1,74 @@
+const tasksSection = document.querySelector('.task-section');
+const taskDesc = document.getElementById('task-value');
+
+let tasks = [];
```

Comment:
- [ ] As required by the project. Please kindly add an array of some simple *to do tasks* (array of objects) with each object containing three keys - description \[`string`\], completed \[`bool`\], index: \[`number`\]

## EXAMPLE
```javascript
const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
];
```

- On page load, the task array should be rendered dynamically to the DOM. The tasks should appear in order of the `index` values for each task.

Reply:
@nati2323 This was not part of the requirements, The hard-coded items should not be removed from the project. I apologize on behalf of @topeogunleye